### DISCUSSION TITLE
Can We Still Be Able To Make Self-Bot or NOT If Yes Then How ?

### DISCUSSION THREAD

**Discussion by CYCNO** [Rapptz/discord.py#8995] [Q&A] [answered]
Can We Still Be Able To Make Self-Bot or NOT If Yes Then How ? 
It Will Be Easier If You Guys Send The Code

**Comment by imvaskel** [comment, marked as answer]
Logging in with user accounts was removed in 2.0 and is additionally against discord TOS.

**Reply by CYCNO** [reply]
Thanks

**Comment by jbaumgarten** [comment]
It's possible but not with that library
